src/components/ComponentDemo/ComponentDemo.js

```diff
@@ -55,7 +58,7 @@ const ComponentDemo = ({
   // allow for write-in props
   // Tests/cleanup context
   // handle true/false better
-
+  console.log('theme', theme, theme.indexOf('g100') > -1);
```

Comment:
Is this left over?

Reply:
yes, actually I'll remove all changes to `ComponentDemo` since they're from an older approach and not needed